File: 02 - Automata.Logic/Automata.WebApi/Parameters/Projects/GetProjectsOut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Automata.WebApi.Entities;

namespace Automata.WebApi.Parameters.Projects
{
    public class GetProjectsOut
    {
        public List<Project> Projects { get; set; }
    }
}

File: 01 - Automata.Presentation/Automata.UI/src/app/classes/method-parameters/projects/getProjectsOut.ts
import { Project } from '../../../entities/entities.module';

export class GetProjectsOut {
    private _projects : Project[];

    get Projects(): Project[] { return this._projects; }
    set Projects(value: Project[]) { this._projects = value; }

    public GetProjectsOut(){
        this._projects = [];
    }
}